settings.py

```diff
@@ -15,21 +16,27 @@
     5432: 'connections',    # PostgreSQL
     5900: 'vnc',            # VNC
     '/var/log/system.log': ('sudo', 'ssh', 'portscan', 'ostiarius'),
+    #'systemd': ('sudo', 'ssh'),
 }
 
 # Enabled output/display methods
 LOGGERS = [
     'stdout',
     'logfile',
-    'osxnotifications',
+    #'osxnotifications',
+    'gnomenotifications',
```

Comment:
Lets stick some OS detection here to determine which loggers to have enabled, I don't want the user to have to modify `settings.py` after downloading to get the app to work on their platform.